pwndbg/gdblib/disasm/arm.py

```diff
@@ -49,9 +49,9 @@
     ARM_INS_STRBT: 1,
     ARM_INS_STRHT: 2,
     ARM_INS_STRT: 4,
-    ARM_INS_STREXB: 1,
-    ARM_INS_STREXH: 2,
-    ARM_INS_STREX: 4,
+    # ARM_INS_STREXB: 1,
+    # ARM_INS_STREXH: 2,
+    # ARM_INS_STREX: 4,
```

Comment:
Why?

Reply:
Fix this in the latest commits and added a test for it - the operands that were accessed to get the source and destination for these were incorrect.